modules/fedimint-ln-common/src/api.rs

```diff
@@ -39,7 +39,7 @@ pub trait LnFederationApi {
     /// Await fedration consensus on a lightning module [`ContractAccount`]
     async fn wait_contract(&self, contract: ContractId) -> FederationResult<ContractAccount>;
     /// Fetch [`OutgoingContractAccount`] for a given contract ID
-    async fn get_outgoing_contract(
+    async fn fetch_outgoing_contract(
```

Comment:
These functions should generally not be used by external software, so changing the verb sounds ok. We don't intend to backport this to 0.2 anyway for lack of functional changes.